### PR TITLE
test(js): remove snapshots

### PR DESCRIPTION
**Pull request by 121watts**
- Removes all snapshot testing
- Adds a linting rule that disallows snapshot testing
- Adds a linting rule that breaks the build if `.only` is used (don't worry you can still run and pass the tests locally)

### CODE REVIEW

File: ui/src/dataExplorer/components/SaveAsButton.test.tsx [hunk 1]
```diff
@@ -16,7 +16,6 @@ describe('SaveAsButton', () => {
   describe('rendering', () => {
     it('renders', () => {
       expect(wrapper.exists()).toBe(true)
```
Comment (drdelambre) [inline ui/src/dataExplorer/components/SaveAsButton.test.tsx]: you deleted all coverage on this component!

File: ui/.eslintrc.js [hunk 2]
```diff
@@ -67,5 +66,7 @@ module.exports = {
     'react/no-unescaped-entities': 'off',
     'react/prop-types': 'off',
     'react/no-find-dom-node': 'off',
+    'jest/no-large-snapshots': ['error', {maxSize: 0}], // no shapshots please
```
Comment (drdelambre) [inline ui/.eslintrc.js]: a bit heavy handed and reactionary

**Comment by drdelambre** [comment]
thanks for removing all of those render snapshots! we can update clockface now!

File: ui/.eslintrc.js (hunk 2 above)
Comment (121watts) [reply ui/.eslintrc.js]: i thought we all agreed no more snapshots.  linter rules make that automatic.

**Pull request merged**